app/src/main/java/fi/mobiilikehitysprojektir13/newsapp/screens/news/components/NewsCategoryChips.kt

```diff
@@ -40,7 +55,8 @@ fun CategoryChips(onChange: (categories: Set<String>) -> Unit) {
                 }
                 onChange.invoke(selectedCategories.toSet())
             }, label = {
-                Text(text = category, color = if (isSelected) Color.Black else Color.White)
+                Text(text = category.replaceFirstChar { if (it.isLowerCase()) it.titlecase() else it.toString() },
```

Comment:
Jos ne on olelisesti jo kaikki lowercase (koska API) niin turhan on tuo if else. Tunge vaan suoran 

```kotlin
Text(text = it.titlecase())
```

Vaikka se ois UPPER_SNAKE_CASE se silti muutu Titlecase:n

Reply:
Should be fixed now to both screens (news screen and article screen)
news screen:
```article?.category?.forEach {category ->
                        AssistChip(onClick = {}, label =  { Text(text = category.replaceFirstChar { it.uppercase() })})```
     article screen:                   
```label = {
                Text(text = category.replaceFirstChar {  it.titlecase()  },```